docs/quick-start.rst

```diff
@@ -34,6 +34,32 @@ context processors are included in this list. For a list of default
 ``TEMPLATE_CONTEXT_PROCESSORS`` please see
 `the official Django docs <https://docs.djangoproject.com/en/1.4/ref/settings/#template-context-processors>`_.
 
+If you are using Django 1.8 then instead of ``TEMPLATE_CONTEXT_PROCESSORS`` you should configure ``TEMPLATES['OPTIONS']['context_processors']``:
+
+.. code-block:: python
+
+    TEMPLATES = [ # example config untill 'context_processors' your config maydiffer
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [
+            ],
+            'APP_DIRS': True,
+            'OPTIONS': {
+                'context_processors': [
+                	# add required context processors here:
+                    'django.core.context_processors.request',
```

Comment:
This has been moved to `django.template.context_processors.request` and this old location is set to be removed in 1.10.